config/startup.options

```diff
@@ -0,0 +1,57 @@
+################################################################################
+# These settings are ONLY used by $LS_HOME/bin/system-install to create a custom
```

Comment:
The JVM options should be in `jvm.options` because Elasticsearch is doing that, and we're doing our best to mirror the configuration profile.  The `jvm.options` _are_ used at each invocation of Logstash, though, as `logstash.lib.sh` is configured to do.  Those can be changed, then restart Logstash to see the changes, where `startup.options` are only for generating init scripts.